lib/project_types/script/messages/messages.rb

```diff
@@ -142,6 +142,9 @@ module Messages
           script_upload_cause: "Fail to upload script.",
           script_upload_help: "Try again.",
 
+          script_too_large_cause: "Script Wasm is too large.",
+          script_too_large_help: "Reduce the size of the Wasm file.",
```

Comment:
@kwringe I'm not sure if this error message is reasonable. It occurs if the user's script Wasm file size is over the limit.

Reply:
>  script_too_large_cause: "Script Wasm is too large.",
>  script_too_large_help: "Reduce the size of the Wasm file.",

Could we be more specific about what we mean by Wasm? Is it the Wasm build? binary?

Reply:
>Could we be more specific about what we mean by Wasm? Is it the Wasm build? binary?

It's the Wasm binary file size for their script.

>Can we specify what the limit is? Is there a specific file size that we support up to?

We should be able to specify the limit by relying on the header value we get. We should have a fallback error message in cases where that header isn't present though.

Reply:
We don't need to say "try again" because this is pretty much implied in every error message. 
What about:

 The size of your Wasm binary file is too large. 
 Make sure that it is less than 30 GB.
 

Other options:
Script can't be pushed because the size of your Wasm build is too large. 
Make sure that your build is less than 30 GB.

Script can't be pushed because the size of your Wasm build is larger than 30 GB. 
Script can't be pushed because the size of your Wasm build exceeds the 30 GB maximum.

Reply:
Yeah, I like that first option. What do we do if the CLI doesn't know the actual limit?

Reply:
We should always provide a number.